src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -86,6 +88,100 @@ public static (bool isSuccess, IConfig config, CommandLineOptions options) Parse
             return result;
         }
 
+        private static IEnumerable<string> ExpandResponseFile(string[] args)
+        {
+            foreach (var arg in args)
+            {
+                if (arg.StartsWith("@"))
+                {
+                    var fileName = arg.Substring(1);
+                    if (File.Exists(fileName))
+                    {
+                        var lines = File.ReadAllLines(fileName);

```

Comment:
Could you please add try..catch with proper exception handling? If there are some issues with the given file (e.g., we don't have permission to read the file content), it would be nice to print an error message rather than just throwing an exception.